Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -114,24 +114,16 @@ struct AssessmentView: View {
                     }
                 }
             }
-            ToolbarItem(placement: .navigationBarLeading) {
-                HStack(alignment: .center) {
-                    Text(exerciseTitle)
-                        .bold()
-                        .font(.title)
-                    Image(systemName: vm.readOnly ? "eyeglasses" : "pencil.and.outline")
-                        .font(.title3)
-                }
-                .foregroundColor(.white)
-            }
-            if cvm.currentlySelecting {
-                ToolbarItem(placement: .navigationBarTrailing) {
-                    Button {
-                        cvm.showAddFeedback.toggle()
-                    } label: {
-                        Text("Feedback")
+            if !vm.readOnly {
+                if cvm.currentlySelecting {
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button {
+                            cvm.showAddFeedback.toggle()
+                        } label: {
+                            Text("Feedback")
+                        }
+                        .disabled(vm.readOnly)
                     }
-                    .disabled(vm.readOnly)
                 }
             }
```

Comment:
This can be removed now, since we implemented the custom feedback Action in EditMenu